styles/style.css

```diff
@@ -539,53 +539,54 @@ textarea::placeholder {
 }
 
 .stak {
-  background-image: url(/pict-mobile/ic_stackoverflow.svg);
+  background-image: url(/Eldonratzlaff.github.io/pict-mobile/ic_stackoverflow.svg);
   background-repeat: no-repeat;
   background-size: contain;
   height: 24px;
   width: 23px;
 }
 
 .git {
-  background-image: url(/pict-mobile/github.svg);
+  background-image: url(/Eldonratzlaff.github.io/pict-mobile/github.svg);
   background-repeat: no-repeat;
   background-size: contain;
   height: 24px;
   width: 23px;
 }
 
 .twitt {
-  background-image: url(/pict-mobile/twitter.svg);
+  background-image: url(/Eldonratzlaff.github.io/pict-mobile/twitter.svg);
   background-repeat: no-repeat;
   background-size: contain;
   height: 24px;
   width: 23px;
 }
 
```

Comment:
- [ ] Seems that there's a problem with your hover functionality since while I'm navigating the webpage your projects appear and disappear from nothing

![image](https://user-images.githubusercontent.com/93630700/211115379-8bfc6e40-176b-4d98-9b68-1586b0ef0e88.png)